src/dialogs/list-diagrams/list-diagrams.tsx

```diff
@@ -27,6 +27,13 @@ export const ListDiagrams: React.FC<Props> = ({
     (state: RootState) => state.dialogPage.diagramInfoDialog.data
   );
 
+  const orderData: any[] = [];
+  const dataNoDefault =
+    data.filter((item: IDiagram) => item.isDefault !== true) ?? [];
+  const dataDefault =
+    data.find((item: IDiagram) => item.isDefault === true) ?? [];
+  orderData.push(...dataDefault, ...dataNoDefault);

```

Comment:
Neste caso, o uso do find e logo em seguida usando spread operator no push vai quebrar a aplicação. Caso o `find` encontre um diagrama que seja default, ele retornará um objeto; quando chegar no push e fizer o spread operator, ele vai quebrar dizendo que precisa ser iterável.

Uma solução seria trocar de find para filter
```suggestion
  const dataDefault =
    data.filter((item: IDiagram) => item.isDefault === true) ?? [];
  orderData.push(...dataDefault, ...dataNoDefault);

```